vectors_handling/uvg.py

```diff
@@ -258,3 +261,85 @@ def generate_top_unigrams(save_path):
 
     top_unigrams = heapq.nlargest(1000, all_unigrams, key=all_unigrams.get)  # fetch top 1000 unigrams
     util.save_file(save_path, top_unigrams)
+
+
+def function_words_process(user_dir):
+    user_vector = [0] * 1000
+
+    for file_dir in os.scandir(user_dir):
+        file = open(file_dir, 'r', encoding='utf-8')
+        lines = file.readlines()
+
+        for line in lines:
+            words = line.split()
+            for word in words:
+                if word in function_words_map.keys():
+                    user_vector[function_words_map.get(word)] += 1
+
+    return user_vector
+
+
+def provide_function_words_map():
+    function_words = provide_function_words()
+
+    global function_words_map
+    function_words_map={}
+
+    for index in range(426):
+        function_words_map[heappop(function_words)] = index
+
+
+def provide_function_words():
```

Comment:
There is no need to "generate" the function words like the other features, those words are being provided so we can directly just load them from a file or a datatype.